src/index.ts

```diff
@@ -105,3 +105,21 @@ export function getCardsV2Json(): Record<string, any>[] {
   }
   return CARDS_JSON;
 }
+
+export function getPrintingsV2Json(): Record<string, any>[] {
+  const PRINTINGS_JSON: Record<string, any>[] = [];
+
+  if (PRINTINGS_JSON.length === 0) {
```

Comment:
This will always be true.

Reply:
yeah, leftover from a misguided caching attempt when i first started this file.  Removed.